### PR TITLE
projects.html: Added GSoC project reports link

### PR DESCRIPTION
**Pull request by KVGarg**
The following commit adds a link of past years **GSoC project reports** in projects.html, just below the project ideas [link](https://projects.coala.io/#/projects) under the search bar.

Fixes https://github.com/coala/projects/issues/683

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: partials/tabs/projects.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
         <input ng-model="searchText" placeholder="Search for a project" id="search" type="search" class="validate"> </div>
       <div style="text-align: center;">
         For more project ideas, <a href="https://github.com/coala/projects/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Project+Proposal%22">click here</a>.
+        <br>
+        For past years <strong>GSoC</strong> project reports, <a href=" http://projects.coala.io/reports">click here</a>.
```
Comment (li-boxuan) [inline partials/tabs/projects.html]: `a href=" http://projects.coala.io/reports"` -> `a href="https://projects.coala.io/reports"`

**Comment by KVGarg** [comment]
@li-boxuan new issues being reported by TravisBuddy , can you suggest me what am I doing wrong?

**Comment by li-boxuan** [comment]
See http://coala.io/git to learn git.

**Comment by KVGarg** [comment]
Do i have to squash the commits?

**Comment by KVGarg** [comment]
Hi @li-boxuan , you  do you mean by squashing the commits ?

**Comment by KVGarg** [comment]
Sorry, closed PR by-mistake :-(

**Comment by KVGarg** [comment]
Hi @li-boxuan , you mentioned rebase the commit properly, do you mean by squashing the commits ?

**Comment by jayvdb** [comment]
Yes, please squash - one commit with a good commit message

**Comment by KVGarg** [comment]
Ok, I will squash the commits :-)

**Comment by KVGarg** [comment]
Successfully managed on squashing the commits 😄

**Comment by NNeast** [comment]
LGTM

**Comment by shashwatmishra76** [comment]
LGTM. 👍

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/685/commits/1eecc141538625612c164cc75f48a24ccb91e609

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

File: partials/tabs/projects.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
         <input ng-model="searchText" placeholder="Search for a project" id="search" type="search" class="validate"> </div>
       <div style="text-align: center;">
         For more project ideas, <a href="https://github.com/coala/projects/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Project+Proposal%22">click here</a>.
+        <br>
+        For past years <strong>GSoC</strong> project reports, <a href="http://projects.coala.io/reports">click here</a>.
```
Comment (KVGarg) [inline partials/tabs/projects.html]: Relative URL was to be used ;-(

**Pull request merged**